frb/builds/build_frbs.py

```diff
@@ -138,8 +139,8 @@ def frb_190102():
                         364.545 * units.pc / units.cm**3,
                         z_frb=0.2912) # Updated redshift
     # Error ellipse [REQUIRED]
-    frb190102.set_ee(0.1, 0.1, 0., 95.) #statistical
-    frb190102.set_ee(0.4, 0.5, 0., 95., stat=False) #systematic
+    frb190102.set_ee(0.16, 0.2, 0., 68.) # Statistical (Day+2020)
+    frb190102.set_ee(0.44, 0.5, 0., 68., stat=False) # Systematic
```

Comment:
Two things:

1.  I calculate 0.46 not 0.44 (and 0.17 not 0.16).  Let's get that resolved
2.  The PA of 0deg is defined in the usual way, i.e.  N.  So you have `a` and `b` backwards.

Reply:
For 0.06s (stat) and 0.16s (sys), and multiplying by 15*cos(-79.4756944), I get = 0.164" and 0.438" (i.e. 0.16 and 0.44).
I have set PA=90 and kept the a and b.